db/migrate/003_add_trello_board_mapping_to_project.rb

```diff
@@ -1,6 +1,6 @@
 class AddTrelloBoardMappingToProject < ActiveRecord::Migration
   def change
-    add_column :projects, :trello_mapping_redmine_statuses, :text, :default => "", :null => false
-    add_column :projects, :trello_mapping_trello_lists,     :text, :default => "", :null => false
```

Comment:
This breaks on PostgreSQL & SQLite if we don't give them a default value.

When I try to reproduce the error on MySQL, the error message is `ActiveRecord::StatementInvalid: Mysql2::Error: BLOB, TEXT, GEOMETRY or JSON column 'trello_mapping_redmine_statuses' can't have a default value: ALTER TABLE `projects` ADD `trello_mapping_redmine_statuses` text DEFAULT ''`.

We need a general way to make a more compatible fix.